src/app/f2f/countryCodes/euDrivingLicence.test.js

```diff
@@ -0,0 +1,7 @@
+const { EU_DL_COUNTRIES } = require("./euDrivingLicence");
+
+describe("#EU_DL_COUNTRIES", () => {
+  it("Should contain nested objects with 2 values - text and code as strings", () => {
+    expect(EU_DL_COUNTRIES.DENMARK).to.deep.equal({ text: "Denmark", code: "DNK"} )
```

Comment:
Snapshot testing https://jestjs.io/docs/snapshot-testing might be a better alternative to just testing one country - what do you think?

Reply:
Could be a good idea!